app/views/needs/_filters.html.erb

```diff
@@ -7,19 +7,22 @@
         <button class="button button--dark filters__controls" aria-expanded="false">Filters</button>
         <div class="filters__content" hidden>
     <% end %>
-        <%= form_with(url: needs_url, method: :get, local: true, class: "filters__form") do %>
+        <%= form_with(url: root_path, method: :get, local: true, class: "filters__form") do |f| %>
```

Comment:
Hm, why change `needs_url` to `root_path`?

Reply:
Because of `link_to_unless_current` we need to use the same as in helper.